trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py

```diff
@@ -141,10 +148,13 @@ def sample_training(self):
             konditions.aerodynamics.angle_of_attack = AoA[i]
             
             # these functions are inherited from Aerodynamics() or overridden
-            CL[i] = calculate_lift_vortex_lattice(konditions, settings, geometry)
+            CL[i], wing_lifts = calculate_lift_vortex_lattice(konditions, settings, geometry)
+            for wing in geometry.wings.values():
```

Comment:
It does look like two now that I open it in an IDE. See my above comment about removing loop.